Index: TLP/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf
===================================================================
--- TLP.orig/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf
+++ TLP/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf
@@ -33,6 +33,12 @@ RADEON_POWER_PROFILE_ON_AC=low
 RADEON_POWER_PROFILE_ON_BAT=low
 RADEON_DPM_STATE_ON_AC=battery
 RADEON_DPM_STATE_ON_BAT=battery
+INTEL_GPU_MIN_FREQ_ON_AC=300
+INTEL_GPU_MIN_FREQ_ON_BAT=300
+INTEL_GPU_MAX_FREQ_ON_AC=500
+INTEL_GPU_MAX_FREQ_ON_BAT=500
+INTEL_GPU_BOOST_FREQ_ON_AC=500
+INTEL_GPU_BOOST_FREQ_ON_BAT=500
 WIFI_PWR_ON_AC=on
 WIFI_PWR_ON_BAT=on
 WOL_DISABLE=Y
Index: TLP/deepin-system-power-control/00-deepin-system-power-control_performance.conf
===================================================================
--- TLP.orig/deepin-system-power-control/00-deepin-system-power-control_performance.conf
+++ TLP/deepin-system-power-control/00-deepin-system-power-control_performance.conf
@@ -31,6 +31,12 @@ RADEON_DPM_PERF_LEVEL_ON_AC=auto
 RADEON_DPM_PERF_LEVEL_ON_BAT=auto
 RADEON_POWER_PROFILE_ON_AC=default
 RADEON_POWER_PROFILE_ON_BAT=default
+INTEL_GPU_MIN_FREQ_ON_AC=300
+INTEL_GPU_MIN_FREQ_ON_BAT=300
+INTEL_GPU_MAX_FREQ_ON_AC=1100
+INTEL_GPU_MAX_FREQ_ON_BAT=1100
+INTEL_GPU_BOOST_FREQ_ON_AC=1100
+INTEL_GPU_BOOST_FREQ_ON_BAT=1100
 WIFI_PWR_ON_AC=off
 WIFI_PWR_ON_BAT=off
 WOL_DISABLE=Y
Index: TLP/deepin-system-power-control/00-deepin-system-power-control_saving.conf
===================================================================
--- TLP.orig/deepin-system-power-control/00-deepin-system-power-control_saving.conf
+++ TLP/deepin-system-power-control/00-deepin-system-power-control_saving.conf
@@ -31,6 +31,12 @@ RADEON_DPM_PERF_LEVEL_ON_AC=low
 RADEON_DPM_PERF_LEVEL_ON_BAT=low
 RADEON_POWER_PROFILE_ON_AC=low
 RADEON_POWER_PROFILE_ON_BAT=low
+INTEL_GPU_MIN_FREQ_ON_AC=300
+INTEL_GPU_MIN_FREQ_ON_BAT=300
+INTEL_GPU_MAX_FREQ_ON_AC=800
+INTEL_GPU_MAX_FREQ_ON_BAT=800
+INTEL_GPU_BOOST_FREQ_ON_AC=800
+INTEL_GPU_BOOST_FREQ_ON_BAT=800
 WIFI_PWR_ON_AC=on
 WIFI_PWR_ON_BAT=on
 WOL_DISABLE=Y
Index: TLP/deepin-system-power-control/00-deepin-system-power-control_balance.conf
===================================================================
--- TLP.orig/deepin-system-power-control/00-deepin-system-power-control_balance.conf
+++ TLP/deepin-system-power-control/00-deepin-system-power-control_balance.conf
@@ -31,6 +31,12 @@ RADEON_DPM_PERF_LEVEL_ON_AC=auto
 RADEON_DPM_PERF_LEVEL_ON_BAT=auto
 RADEON_POWER_PROFILE_ON_AC=default
 RADEON_POWER_PROFILE_ON_BAT=default
+INTEL_GPU_MIN_FREQ_ON_AC=300
+INTEL_GPU_MIN_FREQ_ON_BAT=300
+INTEL_GPU_MAX_FREQ_ON_AC=1100
+INTEL_GPU_MAX_FREQ_ON_BAT=1100
+INTEL_GPU_BOOST_FREQ_ON_AC=1100
+INTEL_GPU_BOOST_FREQ_ON_BAT=1100
 WIFI_PWR_ON_AC=off
 WIFI_PWR_ON_BAT=off
 WOL_DISABLE=Y
