Index: TLP/deepin-system-power-control/00-deepin-system-power-control_balance.conf
===================================================================
--- TLP.orig/deepin-system-power-control/00-deepin-system-power-control_balance.conf	2024-03-06 13:48:11.607596020 +0800
+++ TLP/deepin-system-power-control/00-deepin-system-power-control_balance.conf	2024-03-06 13:51:24.560880006 +0800
@@ -41,7 +41,7 @@
 INTEL_GPU_BOOST_FREQ_ON_BAT=1100
 WIFI_PWR_ON_AC=off
 WIFI_PWR_ON_BAT=off
-WOL_DISABLE=Y
+WOL_DISABLE=N
 SOUND_POWER_SAVE_ON_AC=1
 SOUND_POWER_SAVE_ON_BAT=1
 SOUND_POWER_SAVE_CONTROLLER=Y
Index: TLP/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf
===================================================================
--- TLP.orig/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf	2024-03-06 13:48:11.607596020 +0800
+++ TLP/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf	2024-03-06 13:53:02.721484619 +0800
@@ -43,7 +43,7 @@
 INTEL_GPU_BOOST_FREQ_ON_BAT=500
 WIFI_PWR_ON_AC=on
 WIFI_PWR_ON_BAT=on
-WOL_DISABLE=Y
+WOL_DISABLE=N
 SOUND_POWER_SAVE_ON_AC=1
 SOUND_POWER_SAVE_ON_BAT=1
 SOUND_POWER_SAVE_CONTROLLER=Y
Index: TLP/deepin-system-power-control/00-deepin-system-power-control_performance.conf
===================================================================
--- TLP.orig/deepin-system-power-control/00-deepin-system-power-control_performance.conf	2024-03-06 13:48:11.607596020 +0800
+++ TLP/deepin-system-power-control/00-deepin-system-power-control_performance.conf	2024-03-06 13:56:22.361388863 +0800
@@ -41,7 +41,7 @@
 INTEL_GPU_BOOST_FREQ_ON_BAT=1100
 WIFI_PWR_ON_AC=off
 WIFI_PWR_ON_BAT=off
-WOL_DISABLE=Y
+WOL_DISABLE=N
 SOUND_POWER_SAVE_ON_AC=1
 SOUND_POWER_SAVE_ON_BAT=1
 SOUND_POWER_SAVE_CONTROLLER=Y
Index: TLP/deepin-system-power-control/00-deepin-system-power-control_saving.conf
===================================================================
--- TLP.orig/deepin-system-power-control/00-deepin-system-power-control_saving.conf	2024-03-06 13:48:11.603596317 +0800
+++ TLP/deepin-system-power-control/00-deepin-system-power-control_saving.conf	2024-03-06 13:56:52.195191556 +0800
@@ -41,7 +41,7 @@
 INTEL_GPU_BOOST_FREQ_ON_BAT=800
 WIFI_PWR_ON_AC=on
 WIFI_PWR_ON_BAT=on
-WOL_DISABLE=Y
+WOL_DISABLE=N
 SOUND_POWER_SAVE_ON_AC=1
 SOUND_POWER_SAVE_ON_BAT=1
 SOUND_POWER_SAVE_CONTROLLER=Y
