Index: TLP/func.d/10-tlp-func-cpu
===================================================================
--- TLP.orig/func.d/10-tlp-func-cpu
+++ TLP/func.d/10-tlp-func-cpu
@@ -125,35 +125,62 @@ set_cpu_scaling_governor () {
 
 set_cpu_scaling_min_max_freq () {
     # set CPU scaling limits -- $1: 0=ac mode, 1=battery mode
-    local minfreq maxfreq cpu ec
+    local minfreq maxfreq cpu ec conf_minfreq conf_maxfreq
     local conf=0
 
     if [ "$1" = "1" ]; then
-        minfreq=${CPU_SCALING_MIN_FREQ_ON_BAT:-}
-        maxfreq=${CPU_SCALING_MAX_FREQ_ON_BAT:-}
+        conf_minfreq=${CPU_SCALING_MIN_FREQ_ON_BAT:-}
+        conf_maxfreq=${CPU_SCALING_MAX_FREQ_ON_BAT:-}
     else
-        minfreq=${CPU_SCALING_MIN_FREQ_ON_AC:-}
-        maxfreq=${CPU_SCALING_MAX_FREQ_ON_AC:-}
+        conf_minfreq=${CPU_SCALING_MIN_FREQ_ON_AC:-}
+        conf_maxfreq=${CPU_SCALING_MAX_FREQ_ON_AC:-}
     fi
 
-    if [ -n "$minfreq" ] && [ "$minfreq" != "0" ]; then
+    if [ -n "$conf_minfreq" ] && [ "$conf_minfreq" != "0" ]; then
         conf=1
         ec=0
-        for cpu in "${CPUD}"/cpu*/cpufreq/scaling_min_freq; do
-            if [ -f "$cpu" ] && ! write_sysf "$minfreq" "$cpu"; then
-                echo_debug "pm" "set_cpu_scaling_min_max_freq($1).min.write_error: $cpu $minfreq; rc=$?"
+	#set min freq
+        for cpu in "${CPUD}"/cpu*/cpufreq/; do
+	    #update conf min freq
+            minfreq=$conf_minfreq
+	    #judge min freq
+	    if [ -f "${cpu}/cpuinfo_min_freq" ];then
+	    	actual_min_freq=$(cat "${cpu}/cpuinfo_min_freq")
+		if [ $minfreq -lt $actual_min_freq ];then
+                    minfreq=$actual_min_freq
+                    echo_debug "pm" "set min freq is:$minfreq"
+            	fi
+	    fi
+
+	    write_min_freq="${cpu}/scaling_min_freq"
+
+            if [ -f "${write_min_freq}" ] && ! write_sysf "$minfreq" "${write_min_freq}"; then
+                echo_debug "pm" "set_cpu_scaling_min_max_freq($1).min.write_error: ${write_min_freq} $minfreq; rc=$?"
                 ec=$((ec+1))
             fi
         done
         echo_debug "pm" "set_cpu_scaling_min_max_freq($1).min: $minfreq; ec=$ec"
     fi
 
-    if [ -n "$maxfreq" ] && [ "$maxfreq" != "0" ]; then
+    #set max freq
+    if [ -n "$conf_maxfreq" ] && [ "$conf_maxfreq" != "0" ]; then
         conf=1
         ec=0
-        for cpu in "${CPUD}"/cpu*/cpufreq/scaling_max_freq; do
-            if [ -f "$cpu" ] && ! write_sysf "$maxfreq" "$cpu"; then
-                echo_debug "pm" "set_cpu_scaling_min_max_freq($1).max.write_error: $cpu $maxfreq; rc=$?"
+        for cpu in "${CPUD}"/cpu*/cpufreq/; do
+            #update conf maxfreq
+            maxfreq=$conf_maxfreq
+	    #judge max freq
+	    if [ -f "${cpu}/cpuinfo_max_freq" ];then
+	    	actual_max_freq=$(cat "${cpu}/cpuinfo_max_freq")
+		if [ $maxfreq -gt $actual_max_freq ];then
+	           maxfreq=$actual_max_freq
+    		   echo_debug "pm" "set max freq is:$maxfreq"
+    	    	fi
+	    fi
+
+	    write_max_freq="${cpu}/scaling_max_freq"
+            if [ -f "${write_max_freq}" ] && ! write_sysf "$maxfreq" "${write_max_freq}"; then
+                echo_debug "pm" "set_cpu_scaling_min_max_freq($1).max.write_error: $write_max_freq $maxfreq; rc=$?"
                 ec=$((ec+1))
             fi
         done
Index: TLP/deepin-system-power-control/deepin-system-power-control
===================================================================
--- TLP.orig/deepin-system-power-control/deepin-system-power-control
+++ TLP/deepin-system-power-control/deepin-system-power-control
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+TLP_DIR="/usr/share/tlp/deepin-system-power-control"
+TLP_CONFIG="/etc/tlp.d"
+
 if [ "$(id -u)" != "0" ]; then
     echo "Error:missing root privilege"
     exit 1
@@ -16,16 +19,15 @@ if [ "$2" != "performance" ] && [ "$2" !
 fi
 
 if [ "$2" == "performance" ]; then
-    if [ ! -f "/usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_performance.conf" ]; then
-        echo "File /usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_performance.conf does not exist"
-        exit 1
-    fi
 
-    if ls /etc/tlp.d/00-deepin-system-power-control*.conf 1> /dev/null 2>&1; then
-        rm /etc/tlp.d/00-deepin-system-power-control*.conf
+    if ls ${TLP_CONFIG}/*-deepin-system-power-control*.conf 1> /dev/null 2>&1; then
+       	rm ${TLP_CONFIG}/*-deepin-system-power-control*.conf
     fi
 
-    cp /usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_performance.conf /etc/tlp.d/
+    for conf in $(find ${TLP_DIR} -type f -name *performance.conf);do
+        cp ${conf} ${TLP_CONFIG}/
+    done
+
     if [ $? -ne 0 ]; then
         echo "Copy file failed"
         exit 1
@@ -38,16 +40,15 @@ if [ "$2" == "performance" ]; then
         exit $?
     fi
 elif [ "$2" == "saving" ]; then
-    if [ ! -f "/usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_saving.conf" ]; then
-        echo "File /usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_saving.conf does not exist"
-        exit 1
-    fi
 
-    if ls /etc/tlp.d/00-deepin-system-power-control*.conf 1> /dev/null 2>&1; then
-        rm /etc/tlp.d/00-deepin-system-power-control*.conf
+    if ls ${TLP_CONFIG}/*-deepin-system-power-control*.conf 1> /dev/null 2>&1; then
+        rm ${TLP_CONFIG}/*-deepin-system-power-control*.conf
     fi
 
-    cp /usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_saving.conf /etc/tlp.d/
+    for conf in $(find ${TLP_DIR} -type f -name *saving.conf);do
+        cp ${conf} ${TLP_CONFIG}/
+    done
+
     if [ $? -ne 0 ]; then
         echo "Copy file failed"
         exit 1
@@ -60,16 +61,15 @@ elif [ "$2" == "saving" ]; then
         exit $?
     fi
 elif [ "$2" == "balance" ]; then
-    if [ ! -f "/usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_balance.conf" ]; then
-        echo "File /usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_balance.conf does not exist"
-        exit 1
-    fi
 
-    if ls /etc/tlp.d/00-deepin-system-power-control*.conf 1> /dev/null 2>&1; then
-        rm /etc/tlp.d/00-deepin-system-power-control*.conf
+    if ls ${TLP_CONFIG}/*-deepin-system-power-control*.conf 1> /dev/null 2>&1; then
+        rm ${TLP_CONFIG}/*-deepin-system-power-control*.conf
     fi
 
-    cp /usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_balance.conf /etc/tlp.d/
+    for conf in $(find ${TLP_DIR} -type f -name *balance.conf);do
+        cp ${conf} ${TLP_CONFIG}/
+    done
+
     if [ $? -ne 0 ]; then
         echo "Copy file failed"
         exit 1
@@ -82,16 +82,15 @@ elif [ "$2" == "balance" ]; then
         exit $?
     fi
 elif [ "$2" == "lowbat" ]; then
-    if [ ! -f "/usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf" ]; then
-        echo "File /usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf does not exist"
-        exit 1
-    fi
 
-    if ls /etc/tlp.d/00-deepin-system-power-control*.conf 1> /dev/null 2>&1; then
-        rm /etc/tlp.d/00-deepin-system-power-control*.conf
+    if ls ${TLP_CONFIG}/*-deepin-system-power-control*.conf 1> /dev/null 2>&1; then
+        rm ${TLP_CONFIG}/*-deepin-system-power-control*.conf
     fi
 
-    cp /usr/share/tlp/deepin-system-power-control/00-deepin-system-power-control_lowbat.conf /etc/tlp.d/
+    for conf in $(find ${TLP_DIR} -type f -name *lowbat.conf);do
+        cp ${conf} ${TLP_CONFIG}/
+    done
+
     if [ $? -ne 0 ]; then
         echo "Copy file failed"
         exit 1
