Index: tlp/defaults.conf
===================================================================
--- tlp.orig/defaults.conf	2024-11-22 13:30:44.203413098 +0800
+++ tlp/defaults.conf	2024-11-22 15:28:40.954866880 +0800
@@ -30,8 +30,8 @@
 RADEON_DPM_PERF_LEVEL_ON_BAT=auto
 RADEON_POWER_PROFILE_ON_AC=default
 RADEON_POWER_PROFILE_ON_BAT=default
-AMDGPU_ABM_LEVEL_ON_AC=0
-AMDGPU_ABM_LEVEL_ON_BAT=1
+#AMDGPU_ABM_LEVEL_ON_AC=0
+#AMDGPU_ABM_LEVEL_ON_BAT=1
 WIFI_PWR_ON_AC=off
 WIFI_PWR_ON_BAT=on
 WOL_DISABLE=Y
